### PR TITLE
Fixes Yet ANOTHER King Goat Exploit

### PR DESCRIPTION
**Pull request by fluffe9911**
🆑
bugfix: You can no longer use capsules in the king goat arena
/🆑

Its like whack a mole I tell ya.

### CODE REVIEW

File: code/game/area/areas/ruins/lavaland.dm [hunk 1]
```diff
@@ -37,6 +37,12 @@
 	icon_state = "dk_yellow"
 	ambientsounds = HIGHSEC
 
+/area/ruin/powered/kinggoat_arena //yogs start
```
Comment (Altoids1) [inline code/game/area/areas/ruins/lavaland.dm:40]: Can you dunk this into ``/yogstation/code/game/area/areas/ruins/lavaland.dm``?

**Review by Altoids1 — CHANGES_REQUESTED** [review]

File: code/game/area/areas/ruins/lavaland.dm (hunk 1 above)
Comment (fluffe9911) [reply code/game/area/areas/ruins/lavaland.dm:40]: no cause then it complains a ton about duplicate defs

Comment (Altoids1) [reply code/game/area/areas/ruins/lavaland.dm:40]: Why would it do that...?

Comment (fluffe9911) [reply code/game/area/areas/ruins/lavaland.dm:40]: heck if I know

**Comment by drderp3635** [comment]
This time for sure

**Comment by HomocidlePancakes** [comment]
Don’t mind me, just killing king goat with xray gun

**Comment by fluffe9911** [comment]
Dont think you can reach the king goat with the x ray gun lol

**Comment by HomocidlePancakes** [comment]
I can from space

**Comment by fluffe9911** [comment]
Yes but if someone actually abuses it from space im going to move it to the yogs centcom z level lol

**Comment by HomocidlePancakes** [comment]
Sure. Do it. I will clip out and loot centcomm

**Pull request merged**